src/vivarium_conic_lsff/components/fortification/parameters.py

```diff
@@ -102,6 +102,109 @@ def sample_folic_acid_coverage(location: str, draw: int, coverage_time: str) ->
 
 def sample_folic_acid_relative_risk(location: str, draw: int) -> float:
     seed = get_hash(f'folic_acid_fortification_relative_risk_draw_{draw}_location_{location}')
-    return sample_lognormal_distribution(seed, RELATIVE_RISK)
+    return sample_lognormal_distribution(seed, FOLIC_ACID_FORTIFICATION_RELATIVE_RISK)
 
 
+VITAMIN_A_COVERAGE = {
+    # TODO - numbers for Ethiopia are stubbed
+    'Ethiopia': [
+        {
+            'baseline': BetaParams(
+                upper_bound=1,
+                lower_bound=0,
+                alpha=0.1,
+                beta=9.9,
+            ),
+            'intervention_start': BetaParams(
+                upper_bound=1,
+                lower_bound=0,
+                alpha=0.5,
+                beta=3.1,
+            ),
+            'intervention_end': BetaParams(
+                upper_bound=1,
+                lower_bound=0,
+                alpha=0.8,
+                beta=2.36,
+            ),
+            'weight': 1,
+        },
+    ],
+    'India': [
+        {
+            'baseline': BetaParams.from_statistics(
+                mean=0.243,
+                upper_bound=0.279,
+                lower_bound=0.211,
+            ),
+            'intervention_start': BetaParams.from_statistics(
+                mean=0.894,
+                upper_bound=0.918,
+                lower_bound=0.870,
+            ),
+            'intervention_end': BetaParams.from_statistics(
```

Comment:
Use the normal constructor instead of the class method here with upper_bound=lower_bound=alpha=beta=1

Reply:
Done.